packages/lib/server-only/document/send-document.tsx

```diff
@@ -59,6 +64,10 @@ export const sendDocument = async ({ documentId, userId }: SendDocumentOptions)
         return;
       }
 
+      if (recipient.email === user.email && recipient.name === user.name) {
+        SelfRecipient = true;
+      }
+
```

Comment:
If we don't want to send the email we can just return early here since we do nothing with the Promise return type.

Reply:
I do not want to stop email from bieng sent, but I want to send a specific or default email for self recipient. That is why i am not returning instead I am sending default email according to SelfRecipient variable.